debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -242,6 +242,7 @@ public static BinaryHandlingMode parse(String value, String defaultValue) {
     public static final String DATABASE_CONFIG_PREFIX = "database.";
     private static final String CONVERTER_TYPE_SUFFIX = ".type";
     public static final long DEFAULT_RETRIABLE_RESTART_WAIT = 10000L;
+    public static final long DEFAULT_MAX_QUEUE_SIZE_IN_BYTES = 500 * 1000 * 1024L; // 500MB;
```

Comment:
@jpechane should I set the default value to 0/nagative value, so I can check it to calculate the object size or not?